src/DrugResponseModel.jl

```diff
@@ -56,6 +56,10 @@ export setup_data,
     savitzky_golay_filter,
     optim_all,
     heatmap_combination,
-    plot2
+    plot2,
```

Comment:
You don't need to export every function. In fact, I think it might be better to not export _any_ function, then call them from the Weave files using `DrugResponseModel.function()`.

Reply:
Sure.